### PR TITLE
Created install.bat and changed run.bat

### PR DESCRIPTION
**Pull request by theovit**
Created an installation batch file that checks the Python version installed, creates a Python virtual environment, and installs all prerequisites for the bot. Made adjustments to the run.bat so it interacts with the new install.bat

# Description

1. It checks whether Python version 3.10.x is installed. If not, it guides the user to the installation tutorial.
2. After confirming the correct Python version, it checks for the existence of a pre-existing virtual environment directory. If not found, it creates one.
3. It enters the virtual environment, ensuring an isolated environment for package management.
4. Within the virtual environment, it updates `pip` to the latest version and checks if all specified requirements are installed.
5. If any requirements are missing, the user is prompted to install them. Once installed, it checks for the presence of Playwright and installs it if necessary.
6. The user is given the option to run the bot. If declined, they can choose to rerun the script.

The `run.bat` script complements the `install.bat`:

1. Made a nice little menu with options to install, run the bot, and open a command prompt within the virtual environment.
2. If executed before `install.bat`, it alerts the user that prerequisites are missing and suggests running `install.bat` to install them.
3. If `install.bat` has been run previously, `run.bat` activates the virtual environment and starts the bot.

# Issue Fixes

None

# Checklist:

- [x] I am pushing changes to the **develop** branch
- [x] I am using the recommended development environment
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have formatted and linted my code using python-black and pylint
- [x] I have cleaned up unnecessary files
- [x] My changes generate no new warnings
- [x] My changes follow the existing code-style
- [x] My changes are relevant to the project

# Any other information (e.g how to test the changes)

None

### CODE REVIEW

**Comment by theovit** [comment]
This isn't Python so idk if this applies `I have formatted and linted my code using python-black and pylint`
idk what the existing code style is.... `My changes follow the existing code-style`
I think they are `My changes are relevant to the project`

File: Install.bat [hunk 1]
```diff
@@ -0,0 +1,199 @@
...
+         rmdir /s /q %VENV_DIR%
+    ) else if /i "!choice!"=="N" (
+        call:cecho red "You have chosen not to remove the current virtual environment. Please note this may cause issues with the installation."
+    ) else (
+        call:cecho red "Invalid choice. Please enter Y or N."
+        goto :again
+    )
+)
+cls
+python --version >nul 2>&1
```
Comment (OpenSourceSimon) [inline Install.bat:25]: What if `python` doesn't work but `python3.10` for example does work? Then this script thinks Python isn't installed

File: Install.bat [hunk 2]
```diff
@@ -0,0 +1,199 @@
...
+)
+:: Upgrading pip  
+python -m pip install --upgrade pip >NUL
+if %errorlevel% equ 0 (
+    call:cecho green "Pip is up-to-date."
+) else (
+   call:cecho green "Pip has been upgraded to the latest version."
+)
+:: Check if requirements.txt exists
+if not exist %requirements_file% (
```
Comment (OpenSourceSimon) [inline Install.bat:79]: How can this not exist? Like the user deleted it for some reason??

File: Install.bat [hunk 3]
```diff
@@ -0,0 +1,199 @@
...
+    call:cecho red "%requirements_file% does not exist."
+    exit /b 1
+)
+:: Loop through each line in requirements.txt and check if the package is installed
+echo Checking dependencies...
```
Comment (OpenSourceSimon) [inline Install.bat:84]: pip already does this, you can just do `pip install`

File: Install.bat [hunk 4]
```diff
@@ -0,0 +1,199 @@
...
+    ) else (
+        call:cecho red "Invalid choice. Please enter Y or N."
+        goto :ask_for_choice
+    )
+) else (
+    call:cecho green "All required packages are installed."
+)
+:playwright_check
+pip show playwright >nul 2>&1
+if %errorlevel% equ 0 (
```
Comment (OpenSourceSimon) [inline Install.bat:122]: That doesn't mean that Playwright is fully installed, it needs extra installation: https://reddit-video-maker-bot.netlify.app/docs/installing-playwright

**Review by OpenSourceSimon — CHANGES_REQUESTED** [review]
Good idea, but could you look over the problems?

File: Install.bat (hunk 1 above)
Comment (theovit) [reply Install.bat:25]: oh, so like a PATH issue? I see what you mean. I'll get that resolved. Thanks

File: Install.bat (hunk 2 above)
Comment (theovit) [reply Install.bat:79]: that's a fair point. I guess it is kind of just a useless piece of code, isn't it?

**Comment by theovit** [comment]
The requested changes have been made. I have moved them to a different branch and I will create a pull request to the dev branch. Closing this pull request

**Pull request closed without merging**